testcases/alternativesTests/binaries_test_methods.py

```diff
@@ -105,7 +106,8 @@ def _get_all_binaries_and_slaves(self, pkgs):
                 continue
             if not mexe.DefaultPodman().run_all_scriptlets_for_install(pkg):
                 self.binaries_test.log("Failed to execute postinstall. Slaves will not be found for " + _subpkg)
-            binary_directory_path = self._get_binary_directory_path(name)
+            # binary_directory_path = JVM_DIR + "/" + pkgsplit.get_jvm_dir(name) + "/bin"
+            binary_directory_path = JVM_DIR + "/" + self._get_binary_directory_path(name) + "/bin"
```

Comment:
this looks jsut wrong.

Reply:
you are right.. it should be probably __get_binary_directory_name_ method

Reply:
or I could have added the JVM part in the method itself.. that is also an approach.. probably better